paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -1111,11 +1111,8 @@ internal class PaymentSheetActivityTest {
         val scenario = activityScenario(viewModel)
 
         scenario.launch(intent).onActivity { activity ->
-            testDispatcher.scheduler.advanceTimeBy(50)
-            Espresso.onIdle()
-            assertThat(activity.buyButton.externalLabel?.resolve(context)).isEqualTo("Pay")
```

Comment:
👀 This is safe to delete from the test because the Buy button actually isn't visible on `master`. You can verify this by checking `isVisible` in the test:

![Screenshot 2025-06-03 at 1 22 27 PM](https://github.com/user-attachments/assets/72ee6516-2957-435b-b19f-7d2ff51d2da8)

From a user perspective, while the payment methods are loading we're showing the progress spinner overlay, so content within the sheet isn't visible anyway.